.github/workflows/maven.yml

```diff
@@ -0,0 +1,25 @@
+# This workflow will build a Java project with Maven
+# For more information see: https://help.github.com/actions/language-and-framework-guides/building-and-testing-java-with-maven
+
+name: Java CI with Maven
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up JDK 11
+      uses: actions/setup-java@v2
+      with:
+        java-version: '11'
+        distribution: 'adopt'
+    - name: Build with Maven
+      run: mvn -B package --file pom.xml
```

Comment:
Do we *really* need a Maven workflow? The project is already available at Jitpack, I don't really see a reason why we would need pre-packaged artifacts.

Reply:
Not _really_ no. But it's useful as it tells you when the code won't compile.

Reply:
Also checks PR's. To prevent people from making broken pr's.